spec/spec_helper.rb

```diff
@@ -13,6 +13,7 @@
   end
 end
 
+Grape.config.lint = true # lint all apis by default
```

Comment:
Hmmmm.... I am a bit concerned that we switch a default value in Grape in tests, what are the advantages?

Reply:
I think our CI should always check if Grape is in conformance with Rack's specification. Based on this PR and several CI runs, we can say with a high level of confidence that Grape's is in conformance with Rack 2 and 3 specifications. At least, we do our best by linting our CI constantly and I'm confident that future changes will not break the latter.

Finally, providing the `lint!`feature to everyone without Grape being conformant is not suitable as it should only catch wrongful code from the users.

Reply:
Fun part, while doing it, I found an issue in [rack](https://github.com/rack/rack/issues/2323)

Reply:
Alright, I reluctantly accept this argument.